### ISSUE TITLE
New Map Option in Level editor

### ISSUE THREAD

**Issue by raptor** [bitfighter/bitfighter#248]
_From [Cory.Pou...@gmail.com](https://code.google.com/u/105220164791991617712/) on September 16, 2013 17:19:05_

simple idea you would be able to open a new blank map template for BF and a tab would allow you to cycle what map your working on. also see Load map issue for related content.

_Original issue: http://code.google.com/p/bitfighter/issues/detail?id=248_

**Comment by raptor** [comment]
_From [bkcon...@gmail.com](https://code.google.com/u/112584966209893525339/) on December 05, 2013 10:17:33_

**Status:** Accepted

**Comment by eykamp** [comment]
Maybe ctrl-tab could switch between maps?

**Comment by raptor** [comment]
yeah, CTRL + TAB would make more sense, or maybe CTRL+SHIFT+TAB as there are other CTRL+SHIFT operations (L, S) that have to do with files, I think.

**Comment by eykamp** [comment]
Ctrl+Tab is a common way of cycling windows within an application (on 
Windows, at least), and Ctrl+Shift+Tab does the same, but in reverse order.
